internal/blueprint/blueprint.go

```diff
@@ -48,9 +48,8 @@ type Container struct {
 	Source string `json:"source,omitempty" toml:"source"`
 	Name   string `json:"name,omitempty" toml:"name,omitempty"`
 
-	TLSVerify           *bool   `json:"tls-verify,omitempty" toml:"tls-verify,omitempty"`
```

Comment:
I think the container related changes and the mask changes should be split into separate commits. I also have other concerns about the blueprint field name changes, but since it's already in images that boat may have sailed ;)

Reply:
I had that initially. But then it meant that one of the commits would still be broken and wouldn't compile 🫠 so I squashed them together. Happy to change it back though